cli/src/main.rs

```diff
@@ -51,6 +52,7 @@ pub fn make_cli() -> NewCli<'static> {
         .add_custom::<cmd_instance::CmdInstanceFromImage>("instance from-image")
         .add_custom::<cmd_completion::CmdCompletion>("completion")
         .add_custom::<cmd_timeseries::CmdTimeseriesDashboard>("experimental timeseries dashboard")
+        .add_custom::<cmd_net::CmdNet>("net")
```

Comment:
should this be "system net"? (if not integrated with "system networking")

Reply:
`net` is gone, and I've attempted to integrate the subcommands within the existing hierarchy.